File: URI_1132.cpp
#include <bits/stdc++.h>
 
using namespace std;
 
int main() {
 
    int sum = 0,total=0,i,j,x,y;
        scanf("%d\n",&x);
        scanf("%d\n",&y);
        
        if( x<y )
        {
            for ( i = x; i<= y; i++ )
            {
               if ( i%13 != 0)
               {
                   sum+=i;
               }
            }
            printf("%d\n",sum);
        }
        else {

            for ( j = y; j<= x; j++ )
            {
                if ( j%13 != 0)
                {
                    total+=j;
                }
            }
            printf("%d\n",total);
        }
 
    return 0;
}

File: URI_1132.cs
using System; 

class URI {

    static void Main(string[] args) { 

        int x = int.Parse( Console.ReadLine() );
        int y = int.Parse( Console.ReadLine() );
        int sum = 0,total=0;

        if( x<y )
        {
            for ( int i = x; i<= y; i++ )
            {
               if ( i%13 != 0)
               {
                   sum+=i;
               }
            }
            Console.WriteLine(sum);
        }
        else {

            for ( int i = y; i<= x; i++ )
            {
                if ( i%13 != 0)
                {
                    total+=i;
                }
            }
            Console.WriteLine(total);
        }
        Console.Read();
    }

}
